### PR TITLE
Update 'real' argument for xarray

### PR DESCRIPTION
**Pull request by roxyboy**
I think this should fix issues #78 and #83 .

### CODE REVIEW

**Comment by navidcy** [comment]
@roxyboy, seems to be working fine now.

**Comment by roxyboy** [comment]
Can I merge this @rabernat ?

**Comment by rabernat** [comment]
Sorry for letting this hang for so long without any action.

Every time we fix a bug, we need to add a regression test. So I will merge this as soon as you add a test for the issues raised in #78 and #83.

File: .travis.yml [hunk 1]
```diff
@@ -17,7 +17,7 @@ notifications:
 
 python:
   - 2.7
-  - 3.6
+  - 3.7
```
Comment (rabernat) [inline .travis.yml]: Keep 3.6 please but also add 3.7.

**Comment by navidcy** [comment]
@roxyboy could you please publish a new patch release of the package?

**Pull request merged**